src/lib/util/Config.ts

```diff
@@ -19,23 +18,25 @@ export type EngineConfigContent = {
 	supportedLanguages?: string[];
 }
 
-export const DEFAULT_CONFIG: ConfigContent = {
+const DEFAULT_CONFIG: ConfigContent = {
 	engines: [
 		{
 			name: ENGINE.PMD,
 			targetPatterns: [
 				"**/*.cls","**/*.trigger","**/*.java","**/*.page","**/*.component","**/*.xml",
 				"!**/node_modules/**","!**/*-meta.xml"
 			],
-			supportedLanguages: ['apex', 'vf']
+			supportedLanguages: ['apex', 'vf'],
+			disabled: false
```

Comment:
The current code structure requires all nodes to define something that will be defaulted, or else it risks throwing an exception. It might be nice in 3.0 to change the #lookupAndUpdateToDefault to take a default value instead.

Reply:
Agreed. Back then, modifying `Config.json` seemed like a good idea since the problem back then was porting an older Config to work with the newer Config structure. Right now, the risk of overwriting a file does exist. Incidentally, `vi` editor throws a warning if we try to save a file that was modified in the background by the process.